code/modules/spells/roguetown/seelie.dm

```diff
@@ -0,0 +1,271 @@
+/obj/effect/proc_holder/spell/invoked/seelie_dust
+	name = "Seelie Dust"
+	overlay_state = "createlight"
+	releasedrain = 50
+	charge_max = 150 SECONDS
+	range = 7
+	cast_without_targets = TRUE
+	sound = 'sound/magic/churn.ogg'
+	invocation_type = "none" //can be none, whisper, emote and shout
+
+/obj/effect/proc_holder/spell/invoked/seelie_dust/cast(list/targets, mob/living/user)
+	. = ..()
+	user.emote("giggle")
+	var/target = targets[1]
+	if(isliving(target))
+		var/obj/item/reagent_containers/powder/K = new /obj/item/reagent_containers/powder/SD(get_turf(target))
+		K.reagents.trans_to(target, K.reagents.total_volume, transfered_by = user, method = "swallow")
+		qdel(K)
+	return TRUE
+//Later:tm:
+#if 0
+/obj/effect/proc_holder/spell/targeted/static_room
+	name = "Static"
+	range = 12
+	overlay_state = "necra"
+	releasedrain = 30
+	charge_max = 30 SECONDS
+	max_targets = 0
+	cast_without_targets = TRUE
+	sound = 'sound/magic/churn.ogg'
+	invocation = "*laugh"
+	invocation_type = "none" //can be none, whisper, emote and shout
+
+obj/effect/proc_holder/spell/targeted/static_room/cast(list/targets, mob/user = usr)
```

Comment:
missing a leading slash at the start of the line

Reply:
Im not too sure what this means, the highlighted section for this review (lines 31 to 34) is commented out via a "#if 0" statement on line 21, aka this will not be compiled and executed

Reply:
Nevermind, saw what you meant and resolved it. Should still be syntactically correct even if its commented out for being a WIP